docs/units/equivalencies.rst

```diff
@@ -25,30 +25,28 @@ piece of code needs the same equivalencies, one can set them for a
 Built-in equivalencies
 ======================
 
-Parallax Units
---------------
+How to Convert Parallax to Distance
+-----------------------------------
+
+The length unit *parsec* is defined such that a star one parsec away will exhibit a 1-arcsecond parallax. (Think of it as a contraction between *parallax* and *arcsecond*.)
 
-:func:`~astropy.units.equivalencies.parallax` is a function that returns an
-equivalency list to handle conversions between angles and length.
+The :func:`~astropy.units.equivalencies.parallax` function handles conversions between parallax angles and length.
 
-Length and angles are not normally convertible, so
+In general, one should not be able to change units of length into angles or vice versa, so
```

Comment:
I'm not sure about the "one should". @lglattly , do we have a convention here?

Reply:
Yes, I've been editing any instance of "one must, should, etc." to "you" or "a user" or "the user" depending on the scenario. I think "you should not" works fine here!